apps/app/src/app/pages/(components)/components/(calendar)/calendar--multiple.example.ts

```diff
@@ -0,0 +1,49 @@
+import { Component } from '@angular/core';
+import { HlmCalendarMultiComponent } from '@spartan-ng/ui-calendar-helm';
+
+@Component({
+	selector: 'spartan-calendar-multiple',
+	imports: [HlmCalendarMultiComponent],
+	template: `
+		<hlm-calendar-multi [(date)]="selectedDate" [min]="minDate" [max]="maxDate" [minSelection]="2" [maxSelection]="6" />
+	`,
+	host: {
+		class: 'preview flex min-h-[350px] w-full justify-center p-10 items-center',
+	},
+})
+export class CalendarMultipleExampleComponent {
+	/** The selected date */
+	public selectedDate = [new Date()];
```

Comment:
Could this be `selectedDates` just to indicate that it is can be multiple dates.

Reply:
Yes of course!